app/data/data.go

```diff
@@ -0,0 +1,17 @@
+package data
+
+import (
+	"fmt"
+)
+
+// Data is data.
+type Data struct {
+	ID string
+}
+
+func (d Data) String() string {
+	if d.ID == "" {
+		panic("empty ID")
```

Comment:
**[self-lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
forbidden to write panic